README.md

```diff
@@ -9,36 +9,104 @@
 | Rayce Rossum | [RayceRossum](https://github.com/RayceRossum) |
 
 
-### Overview
+## Overview
 
-A common application of supervised machine learning is identifying the object of an image. One issue that users encounter is a model misclassifying a new image because the object is rotated or translated in some way that was not captured in the training images. The purpose of this package is to create a more robust set of images for users to train their model with. The package will accept an image as an input, apply a series of translations to it, as specified by the user, and return an array of translated pixel values. Translations include: rotating, mirroring, and translating (shifting the object's location in the frame).
+A common application of supervised machine learning is identifying the object of an image. One issue that users encounter is a model misclassifying a new image because the object is rotated or translated in some way that was not captured in the training images. The purpose of this package is to create a more robust set of images for users to train their model with. The package will accept an image as an input, apply a series of transformations to it, and return an array of transformed pixel values. Transformations include: rotating, mirroring, and translating (shifting the object's location in the frame).
 
-### Functions
 
-##### Rotate
-- Parameters:
-  - Image to transform (string: PATH)
-  - Number of images to generate (integer)
-  - Maximum rotation in degrees (integer)
-- Returns:
-  - Array of randomly rotated (within max range) images of length based on number of images parameter
+## Functions
 
-##### Mirror
-- Parameters:
-  - Image to transform (string: PATH)
-  - Direction of mirroring (string: 'horizontal', 'vertical', 'both')
-- Returns:
-  - Array of mirrored image(s)
+#### Rotate
 
-##### Translate
-- Parameters:
-  - Image to transform (string: PATH)
-  - Number of images to generate (integer)
-  - Maximum distance in pixels (integer)
-- Returns:
-  - Array of randomly translated (within max range) images of length based on number of images parameter
+Rotates an image the user-specified number of times. The degree of rotation is chosen randomly and may be in either the clockwise or counter-clockwise direction. The user specifies the maximum rotation angle. It returns the pixel values of the rotated images.
```

Comment:
"The degree of rotation is chosen randomly and may be in either the clockwise or counter-clockwise direction" -> need to update because the rotation in R will only be clockwise.

Reply:
Fixed the wording for rotate